diff -U4 -r lazarus-3.6--orig/lazarus/lcl/interfaces/qt5/qtwsforms.pp lazarus-3.6--patched/lazarus/lcl/interfaces/qt5/qtwsforms.pp
--- lazarus-3.6--orig/lazarus/lcl/interfaces/qt5/qtwsforms.pp	2024-09-29 11:39:45.000000000 +0200
+++ lazarus-3.6--patched/lazarus/lcl/interfaces/qt5/qtwsforms.pp	2024-12-09 16:58:22.751428297 +0100
@@ -1034,20 +1034,23 @@
     begin
       if Assigned(TCustomForm(AWinControl).Menu) then
       begin
         AWin := TQtMainWindow(AWinControl.Handle);
-        AWin.MenuBar.sizeHint(@ASize);
-        // geometry isn't updated yet
-        if ASize.cy < 10 then
-          ASize.cy := 0;
-        // we must use real geometry, and then exclude menubar height.
-        aClientRect := AWin.getGeometry;
-        Types.OffsetRect(aClientRect, -aClientRect.Left, -aClientRect.Top);
-        dec(AClientRect.Bottom, ASize.cy);
-        {$IFDEF VerboseQtResize}
-        DebugLn('TQtWSCustomForm.getDefaultClientRect ',dbgsName(AWinControl),' ',dbgs(AWin.getClientBounds),' mnuBarHeight ',dbgs(AWin.MenuBar.getHeight),' ASize=',dbgs(ASize),' FINAL=',dbgs(AClientRect));
-        {$ENDIF}
-        Result := True;
+        if Assigned(AWin.MenuBar) then
+        begin
+          AWin.MenuBar.sizeHint(@ASize);
+          // geometry isn't updated yet
+          if ASize.cy < 10 then
+            ASize.cy := 0;
+          // we must use real geometry, and then exclude menubar height.
+          aClientRect := AWin.getGeometry;
+          Types.OffsetRect(aClientRect, -aClientRect.Left, -aClientRect.Top);
+          dec(AClientRect.Bottom, ASize.cy);
+          {$IFDEF VerboseQtResize}
+          DebugLn('TQtWSCustomForm.getDefaultClientRect ',dbgsName(AWinControl),' ',dbgs(AWin.getClientBounds),' mnuBarHeight ',dbgs(AWin.MenuBar.getHeight),' ASize=',dbgs(ASize),' FINAL=',dbgs(AClientRect));
+          {$ENDIF}
+          Result := True;
+        end;
       end;
     end;
   end;
 end;
