--- transgui.lpi.orig	2025-06-01 17:50:09.816938126 -0400
+++ transgui.lpi	2025-06-01 18:09:32.078881519 -0400
@@ -274,6 +274,7 @@
           <PathDelim Value="\"/>
           <SearchPaths>
             <IncludeFiles Value="synapse\source\lib"/>
+            <Libraries Value="\usr\lib\gcc\x86_64-linux-gnu\14"/>
             <OtherUnitFiles Value="synapse\source\lib"/>
             <UnitOutputDirectory Value="units"/>
           </SearchPaths>
@@ -537,6 +538,7 @@
     <PathDelim Value="\"/>
     <SearchPaths>
       <IncludeFiles Value="synapse\source\lib"/>
+      <Libraries Value="\usr\lib\gcc\x86_64-linux-gnu\14"/>
       <OtherUnitFiles Value="synapse\source\lib"/>
       <UnitOutputDirectory Value="units"/>
     </SearchPaths>
